.github/workflows/eks-addon-tests.yaml

```diff
@@ -0,0 +1,34 @@
+name: EKS Add-on Tests
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
+
+jobs:
+  eks-addon-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # TODO: Use ci-matrix.json instead
```

Comment:
It should be pretty simple to just copy+paste the existing [`get-test-matrix` job from other workflows](https://github.com/signalfx/splunk-otel-collector-chart/blob/4e864ab1ae7b1b9cdeaa7925a9c4603e0d9ef81c/.github/workflows/functional_test.yaml#L23), can we just do it now? The follow up could be to move to a reusable job so it's in a central place.

Reply:
It's not only that. We need to also update the update the script that keep the matrix up to date. I'd move that to follow up

Reply:
Currently we have particular group per specific target. Another problem as not all of them are being watched and updated e.g. I'd like to simplify and clean it up separately, then adopt here